internal/mq/msgstream/mq_msgstream.go

```diff
@@ -841,30 +841,11 @@ func (ms *MqTtMsgStream) allChanReachSameTtMsg(chanTtMsgSync map[mqwrapper.Consu
 
 // Seek to the specified position
 func (ms *MqTtMsgStream) Seek(msgPositions []*internalpb.MsgPosition) error {
-	var consumer mqwrapper.Consumer
 	var mp *MsgPosition
-	var err error
-	fn := func() error {
-		var ok bool
-		consumer, ok = ms.consumers[mp.ChannelName]
-		if !ok {
-			return fmt.Errorf("please subcribe the channel, channel name =%s", mp.ChannelName)
-		}
-
-		if consumer == nil {
-			return fmt.Errorf("consumer is nil")
-		}
 
-		seekMsgID, err := ms.client.BytesToMsgID(mp.MsgID)
-		if err != nil {
-			return err
-		}
-		err = consumer.Seek(seekMsgID, true)
-		if err != nil {
-			return err
-		}
-
-		return nil
+	fn := func() error {
+		// seek by MsgID logic, same to mqMsgStream
+		return ms.mqMsgStream.Seek([]*MsgPosition{mp})
```

Comment:
Is it OK to calling Seek on internal mqMsgStream instead of the corresponding consumer?

Reply:
Here, yes, we use channel name in `MsgPosition` to locate consumer and then do seek on the consumer.
I also compared the logic of `fn` here and `mqMsgStream.Seek`, it's same.